subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -173,8 +173,15 @@ enum node_rx_type {
 	NODE_RX_TYPE_MESH_ADV_CPLT = 0x13,
 	NODE_RX_TYPE_MESH_REPORT = 0x14,
 #endif /* CONFIG_BT_HCI_MESH_EXT */
-};
 
+/* Following proprietary defines must be at end of enum range */
+#if defined(CONFIG_BT_CTLR_USER_EXT)
+	NODE_RX_TYPE_USER_START = 0x15,
```

Comment:
why 0x15? this is completely arbitrary? should it be configurable?

Reply:
This is at the end of the other constant in the enum, for some reason they have all been assigned numbers, so I'm just following the prior art here. Keeping the constants ordered makes sense if someone wants to use them as an index in a lookup table, a wide spread of numbers would result in a very sparse table structure

Reply:
The idea behind explicit enum values is to simplify debug, e.g. tracing.
Also, we are sure they don't change between different builds -- delaying rot of traces :)

Reply:
I will leave this as is

Reply:
I did implement the suggested Kconfig change (depends on BT_CTLR_USER_EXT)